res/CircleRecipes/RoPE_000/test.recipe

```diff
@@ -0,0 +1,70 @@
+operand {
+  name: "ifm"
+  type: FLOAT32
+  shape { dim: 1 dim: 1 dim: 1 dim: 80 }
```

Comment:
can you plz add a comment why 80 real number items exist?

Reply:
i updated and i think it is hard to explain what must be 80 real number.
it is composed of the same conditions by referring to the model using the rope.
so. if there is no load on the test environment, i think it would be better to maintain the value